blog/templates/blog/_blog_card.html

```diff
@@ -9,17 +9,13 @@
 					alt="{{ x1.alt }}"
 					src="{{ x1.url }}"
 				>
-			{% elif photo.0.block_type == "vertical_bar_chart" %}
+			{% elif photo.0.block_type == "vertical_bar_chart" or photo.0.block_type == "tree_map_chart" or photo.0.block_type == "bubble_map_chart" %}
 				<div class="card-image card-image-{{ size }}">
-					{% include 'common/blocks/vertical_bar_chart.html' with value=photo.0.value mini_graphic=true static_graphic=true %}
-				</div>
-			{% elif photo.0.block_type == "tree_map_chart" %}
-				<div class="card-image card-image-{{ size }}">
-					{% include 'common/blocks/tree_map_chart.html' with value=photo.0.value mini_graphic=true static_graphic=true %}
-				</div>
-			{% elif photo.0.block_type == "bubble_map_chart" %}
-				<div class="card-image card-image-{{ size }}">
-					{% include 'common/blocks/bubble_map_chart.html' with value=photo.0.value mini_graphic=true static_graphic=true %}
+					<img
+						class="card-image card-image-{{ size }}"
+						alt="{{ photo.0.value.description }}"
+						src="{{ photo.0.value.svg_snapshot_mini_datauri }}"
```

Comment:
In general we tend to use inline SVGs over image tags for SVG rendering, so might be something to consider revising in the future. I can't think of any concrete advantages either way at the moment though, so fine to leave as-is.